src/fromager/commands/build.py

```diff
@@ -93,24 +117,42 @@ def build_sequence(
             wkctx.wheel_server_url,
         )
 
+    entries: list[BuildSequenceEntry] = []
+
     logger.info("reading build order from %s", build_order_file)
     with open(build_order_file, "r") as f:
         for entry in progress.progress(json.load(f)):
             dist_name = entry["dist"]
             resolved_version = Version(entry["version"])
+            prebuilt = entry["prebuilt"]
             source_download_url = entry["source_url"]
+
             req = Requirement(f"{dist_name}=={resolved_version}")
 
-            if skip_existing and _is_wheel_built(wkctx, dist_name, resolved_version):
-                logger.info(
-                    "%s: skipping building wheels for %s==%s since it already exists",
-                    dist_name,
-                    dist_name,
-                    resolved_version,
+            if skip_existing:
+                is_built, wheel_filename = _is_wheel_built(
+                    wkctx, dist_name, resolved_version
                 )
-                continue
-
-            if entry["prebuilt"]:
+                if is_built:
+                    logger.info(
+                        "%s: skipping building wheels for %s==%s since it already exists",
+                        dist_name,
+                        dist_name,
+                        resolved_version,
+                    )
+                    entries.append(
+                        BuildSequenceEntry(
+                            dist_name,
+                            resolved_version,
+                            prebuilt,
+                            source_download_url,
+                            wheel_filename=wheel_filename,
+                            skipped=True,
+                        )
+                    )
```

Comment:
We don't want to include the skipped wheels in the summary. The idea was to use the summary as part of our changelog in the releases we make downstream

Reply:
This requirement was not obvious to me. How about two tables, one with rebuild packages and one with packages that have been retained from a previous build?

Reply:
That works as well

Reply:
I'm now printing three tables: build, pre-built, and skipped.